Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -1962,7 +1962,23 @@
     <!-- XACT: Button/Dialog label to cancel something-->
     <string name="generic_action_abort">"Cancel"</string>
     <!-- XACT: Button/Dialog label to remove something-->
-    <string name="generic_action_remove">"Remove"</string>
+    <string name="generic_action_remove">Remove</string>
+
+    <!-- ####################################
+        Incompatibility warning card
+    ###################################### -->
+    <string name="incompatible_headline">Incompatibility warning</string>
+    <string name="incompatible_advertising_not_supported">Unfortunately, your device is only
+        partially compatible with Bluetooth-based Exposure Notifications. You can be notified of
+        risk contacts but won\'t be able to warn users outside of locations and events you have
+        checked in to.
+    </string>
+    <string name="incompatible_scanning_not_supported">Unfortunately, your device is not compatible
+        with Bluetooth-based Exposure Notifications. Only check-In functionality is supported on
+        your device.
+    </string>
+
+    <string name="incompatible_link">"https://www.coronawarn.app/en/faq/#incompatibility_warning"</string>
```

Comment:
@chiljamgossow we need german strings, english strings in this file won't be recognized by the translation system. Please sync with @janetback for the correct strings.

Reply:
We want German in the English string file?

Reply:
No, we don't provide english texts this come from our translators. We provide german texts and then get the english ones. To satisfy our Linter we still need the empty strings here or you can put the germans one in (to not cause any crashes) after you also added them to the german string file. The same process as for all other strings applies here.

Reply:
removed